### PR TITLE
Add log4j properties 

### PR DESCRIPTION
**Pull request by RoccoDiGirolamo**
Changes done:

1. New template and tasks to configure log4j.properties. The new setting for log4j allows us to manage the logs (in terms of location/cleanup policy/ log name/ log level).
2. New kafka and zookeeper environment templates. The old kafka-jvm.sh became kafka-env.sh. In this way we can also add other environment variables to these files. A default setting in them is the _KAFKA_LOG4J_OPTS_ to be able to set the new log config file. There is also an empty var that could be set if we need another optional environement var.
3. I made optionals the creation of the link to the log folder as we can change the location of the logs thanks to the Log4j properties
4. Updated the services file to be able to use the environment script and to be able to set the _timeoutStartSec_ (default value set to 90 secs as in systemd documentation)

### CODE REVIEW

File: defaults/main.yaml [hunk 1]
```diff
@@ -161,3 +161,69 @@ zookeeper_services:
 ## Ensure our topic does no longer exist if it is in the topic list
 #  - name: unused_topic
 #    state: absent
+
+#Log4J Vars
+log4j_logDir: "{{ dst_path }}/confluent-{{ confluent_version }}/logs"
```
Comment (macunha1) [inline defaults/main.yaml]: Sorry for nitpicking about this. Could you change the variable names to snake case? To follow the Ansible code style

File: defaults/main.yaml [hunk 2]
```diff
@@ -161,3 +161,69 @@ zookeeper_services:
...
+log4j_additivity_kafka_logCleaner: "false"
+
+log4j_logger_kafka_stateChangeLogger: "TRACE, stateChangeAppender"
+log4j_additivity_kafka_stateChangeLogger: "false"
+
+log4j_logger_kafka_authorizerLogger: "INFO, authorizerAppender"
+log4j_additivity_kafka_authorizerLogger: "false"
+
+#Service Vars
+env_var: ""
```
Comment (macunha1) [inline defaults/main.yaml]: As the default is empty, we could skip the `env_var` from defaults.

Would you mind changing the name to something less ambiguous? Maybe `additional_profile_env_vars`, to pass the idea that is going to be appended to the default (JVM) ones

File: templates/kafka-env.sh.j2 [hunk 3]
```diff
@@ -3,3 +3,5 @@ KAFKA_JVM_PERFORMANCE_OPTS="-server -XX:MetaspaceSize=96m -XX:+UseG1GC -XX:MaxGC
 {% if (memory_heapsize|int) > 1 %}
 KAFKA_HEAP_OPTS="-Xmx{{ memory_heapsize }}g -Xms{{ memory_heapsize }}g"
 {% endif %}
+KAFKA_LOG4J_OPTS="-Dlog4j.configuration=file:{{ conf_dest }}/log4j.properties"
+{{ env_var }}
```
Comment (macunha1) [inline templates/kafka-env.sh.j2]: Considering that we remove `env_var` from the `defaults/main.yaml` it is then possible to use ` | default(omit)` in order to keep the output clean.

e.g.:

```
{{ env_var | default(omit) }}
```

File: templates/zookeeper-env.sh.j2 [hunk 4]
```diff
@@ -0,0 +1,2 @@
+KAFKA_LOG4J_OPTS="-Dlog4j.configuration=file:{{ conf_dest }}/log4j.properties"
+{{ env_var }}
```
Comment (macunha1) [inline templates/zookeeper-env.sh.j2]: Same suggestion for `default(omit)` applies here

```
{{ env_var | default(omit) }}
```

**Review by macunha1 — CHANGES_REQUESTED** [review]
Hello!

I liked your idea of setting the `log4j` as it provides more convenience during the Kafka logging setup. Thanks for your contribution.

Could you please take a look at the review and fix the [reported issues from CircleCI](https://app.circleci.com/pipelines/github/macunha1/confluent-kafka-role/474/workflows/60942ccb-05c1-466a-ab25-a5c6705beefe/jobs/4479)?

File: defaults/main.yaml (hunk 1 above)
Comment (RoccoDiGirolamo) [reply defaults/main.yaml]: Yeah, you're right. I changed all variables to snake case

File: defaults/main.yaml (hunk 2 above)
Comment (RoccoDiGirolamo) [reply defaults/main.yaml]: Done. I deleted the var from defaults, changed the name and added `| default(omit)`

File: templates/kafka-env.sh.j2 (hunk 3 above)
Comment (RoccoDiGirolamo) [reply templates/kafka-env.sh.j2]: Done. I deleted the var from defaults, changed the name and added `| default(omit)`

**Review by macunha1 — APPROVED** [review]

**Pull request merged**